alchemiscale/tests/integration/storage/test_statestore.py

```diff
@@ -270,6 +273,92 @@ def test_get_transformation_failures(
 
     ### compute
 
+    def test_register_computeservice(self, n4js, compute_service_id):
+        now = datetime.utcnow()
+        registration = ComputeServiceRegistration(
+            identifier=compute_service_id, registered=now, heartbeat=now
+        )
+
+        n4js.register_computeservice(registration)
+
+        csreg = n4js.graph.run(
+            f"""
+            match (csreg:ComputeServiceRegistration {{identifier: '{compute_service_id}'}})
+            return csreg
+            """
+        ).to_subgraph()
+
+        assert csreg["identifier"] == compute_service_id
+        assert csreg["registered"] == now
+        assert csreg["heartbeat"] == now
+
+    def test_deregister_computeservice(self, n4js, compute_service_id):
+        now = datetime.utcnow()
+        registration = ComputeServiceRegistration(
+            identifier=compute_service_id, registered=now, heartbeat=now
+        )
+
+        n4js.register_computeservice(registration)
```

Comment:
test return values from database queries here also ?

Reply:
Ah, good catch; will add this.